scripts/audit/find_real_orphans.py

```
@@ -0,0 +1,379 @@
+    total_lines = 0
+    for o in orphans:
+        rel = o.relative_to(app_root)
+        lines = wc_l(o)
+        groups[categorize(rel)].append((rel, lines))
+        total_lines += lines
+
+    # 6. 生成报告
+    lines_out: list[str] = []
+    lines_out.append("# Audit · 真孤儿证伪报告 (脚本自动生成)")
+    lines_out.append("")
+    lines_out.append(f"- 脚本: `scripts/audit/find_real_orphans.py`")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)